Packs/Endace/ReleaseNotes/1_2_0.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Endace Search Archive Download PCAP v3
+
+- %%UPDATE_RN%%
```

Comment:
Please update the release notes according to the changes/additions.

Reply:
Updated